app/webpacker/components/MyCompetitions/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import {
+  Accordion,
+  Header,
+  Icon,
+  Checkbox, Segment,
+} from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import {
+  personUrl,
+} from '../../lib/requests/routes.js.erb';
+import UpcomingCompetitionTable from './UpcomingCompetitionTable';
+import PastCompetitionsTable from './PastCompetitionTable';
+
+export default function MyCompetitions({ permissions, competitions, userInfo }) {
+  const [isAccordionOpen, setIsAccordionOpen] = useState(false);
+  const [shouldShowRegistrationStatus, setShouldShowRegistrationStatus] = useState(false);
+
+  return (
+    <>
+      <Header>
+        {I18n.t('competitions.my_competitions.title')}
+      </Header>
+      <p>
+        {I18n.t('competitions.my_competitions.disclaimer')}
+      </p>
+      <UpcomingCompetitionTable
+        competitions={competitions.futureCompetitions}
+        permissions={permissions}
+        registrationStatuses={competitions.registrationStatuses}
+      />
+      <Accordion fluid styled>
+        <Accordion.Title
+          active={isAccordionOpen}
+          onClick={() => setIsAccordionOpen(!isAccordionOpen)}
```

Comment:
I guess the more React-esque way would be `() => setIsAccordionOpen((prevValue) => !prevValue)` but this is not a hill I'll die on.